Userland/Utilities/nc.cpp

```diff
@@ -192,10 +217,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         }
 
         if (!fd_closed && FD_ISSET(fd, &readfds)) {
-            Array<u8, 1024> buffer;
-            Bytes buffer_span = buffer.span();
-            auto nread = TRY(Core::System::read(fd, buffer_span));
-            buffer_span = buffer_span.trim(nread);
+            auto buffer = AK::ByteBuffer::create_zeroed(rcv_buff_size)->bytes();
```

Comment:
There are two problems here:
- The way the code is written you will have a use after free after this line returns, as the backing `ByteBuffer` will be destroyed, and you will be left with the `Bytes` pointing to bogus memory. 
- You need to wrap the `create_zeroed` with `TRY(..)`

```suggestion
            auto buffer = TRY(AK::ByteBuffer::create_zeroed(rcv_buff_size)));
```